kiali/README.md

```diff
@@ -40,22 +40,38 @@ An alternative can be a parallel installation of the upstream chart offering all
 
 1. Run the Helm upgrade command, which installs the chart if not present yet.
     ```bash
-    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.auth.strategy=anonymous -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
+    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
     ```
 
 You can either use the `[values.yaml](./values.yaml)` provided in this `kiali` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
 
 ### Verify the installation
 
-1. You should see the `kiali-operator` and `kiali` Pod coming up in the Namespace. All Pods must eventually be in a "Running" state.
-1. Browse the Kiali dashboard. The following command exposes the dashboard on `http://localhost:20001`:
-   ```bash
-   kubectl -n ${KYMA_KIALI_NS} port-forward svc/kiali-server 20001
-   ```
+You should see the `kiali-operator` and `kiali-server` Pod coming up in the Namespace. All Pods must eventually be in a "Running" state.
+
+### Access Kiali
+
+To access Kiali, you can either use kubectl port forwarding, or expose it using the Kyma ingress gateway.
+
+* To access Kiali using port forwarding, run:
+  ```bash
+  kubectl -n ${KYMA_KIALI_NS} port-forward svc/kiali-server 20001
+  ```
+
+  Open Kiali in your browser under [http://localhost:20001](http://localhost:20001) and login with a [Kubernetes service account token](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens), for instance from your kubeconfig file.
+
+* To expose Kiali using the Kyma API gateway, create an APIRule:
+  ```bash
+  kubectl -n ${KYMA_KIALI_NS} apply -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
```

Comment:
Pardon me, I forgot that it works with or without the curly brackets when I added the comment. It must have been because of the other error I got that I just connected the wrong dots. 
But either way, without the brackets is the convention we use in Kyma documentation for environment variables, and we use the brackets for placeholders, so it's still good that you removed them. I just wanted to correct myself :)